app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/initial/InitialActivity.kt

```diff
@@ -53,9 +54,19 @@ class InitialActivity : DaggerAppCompatActivity() {
             "9,6,000000001000010000000010000100001000,noboru",
             "10,6,000100000010010000000100000010010000,noboru"
         )
-        for (data in initData) {
-            tumeKyouenRepository.insertByCSV(data).blockingAwait()
+        val stages = initData.map {
+            val items = it.split(",")
+            TumeKyouen(
+                0,
+                Integer.parseInt(items[0]),
+                Integer.parseInt(items[1]),
+                items[2],
+                items[3],
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/kyouen-android/blob/a5c24ec48abc4b18a58802172ea3f43c28a46200/app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/initial/InitialActivity.kt#L64"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>